homeassistant/components/sensor/dnsip.py

```diff
@@ -0,0 +1,100 @@
+"""
+Get your own public IP address or that of any host.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.dnsip/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.const import STATE_UNKNOWN
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import Throttle
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['dnspython==1.15.0']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_HOSTNAME = 'hostname'
+CONF_RESOLVER = 'resolver'
+CONF_RESOLVER_IPV6 = 'resolver_ipv6'
+CONF_IPV6 = 'ipv6'
+
+DEFAULT_HOSTNAME = 'myip.opendns.com'
+DEFAULT_RESOLVER = '208.67.222.222'
+DEFAULT_RESOLVER_IPV6 = '2620:0:ccc::2'
+DEFAULT_IPV6 = False
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=120)
```

Comment:
Use `SCAN_INTERVAL` instead. That allow the user also on config to set the update interval